### ISSUE TITLE
Add IPAM release/show IP commands to calicoctl

### ISSUE THREAD

**Issue by robbrockbank** [projectcalico/libcalico-go#87]

**Comment by gunjan5** [comment]
Behavior changes from the python version: 

`calicoctl ipam info <IP>` —> `calicoctl ipam show <IP>` 

Python calicoctl:
```bash
vagrant@calico-01:~$ calicoctl ipam info 192.168.142.1
No attributes defined for 192.168.142.1
vagrant@calico-01:~$ calicoctl ipam info 192.168.142.4
IP 192.168.142.4 is not currently assigned
```

Go calicoctl:
```bash 
vagrant@calico-01:~$ calicoctl ipam show 192.168.142.1
No attributes defined for IP 192.168.142.1
vagrant@calico-01:~$ calicoctl ipam show 192.168.142.4
IP 192.168.142.4 is not currently assigned in block
```

**Comment by gunjan5** [comment]
fixed in #145
